README.md

```diff
@@ -81,6 +81,24 @@ fn main() {
 }
 ```
 
+## Features
+
+`async-std` is strongly commited to following semver. This means your code won't
+break unless _you_ decide to upgrade.
+
+However every now and then we come up with something that we think will work
+_great_ for `async-std`, and we want to provide a sneak-peek so you can try it
+out. This is what we call _"unstable"_ features. You can try out the unstable
+features by enabling the `unstable` feature in you `Cargo.toml` file:
+
+```toml
+[dependencies]
+async-std = { version = "0.99.5", features = ["unstable"] }
```

Comment:
In #190, I used the following syntax:

```toml
[dependencies.async-std]
version = "0.99"
features = ["unstable"]
```

Do you have a preference? I personally don't so pick whichever you like best. :)

Reply:
```suggestion
[dependencies.async-std]
version = "0.99"
features = ["unstable"]
```